src/search/search-index/add.js

```diff
@@ -180,3 +180,31 @@ async function performIndexing(indexDoc) {
         console.error(err)
     }
 }
+
+async function indexingMetaConcurrent(pageId, timestamp) {
+    const reverseIndexDoc = await singleLookup(pageId)
+    reverseIndexDoc.visits.add(timestamp)
+    await index.put(pageId, {
```

Comment:
Can just call `await index.put(pageId, reverseIndexDoc)` here. Using destructuring here will just create a new object from the contents of `reverseIndexDoc` before storing, but I don't think that's needed.

Reply:
I would also name it something like `updateReverseIndexMeta` or something (no need for "concurrent" as this isn't interfacing with the queue in this scope) to better indicate what it does. Also it should take a `type` parameter, similar to `addMetaConcurrent`, and decide which `Set` to update from that, as now it will only work for visits. Also either assume `timestamp` parameter already has the meta prefix (`'visit/'` or `'bookmark/'`) or prepend it before calling `.add(timestamp)`, as these `Set`s are meant to contain references to other index keys (now it only references the timestamp which is just part of the key)